broker/setup.py

```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 from setuptools import find_packages, setup
 
-VERSION = '0.3.0'
+VERSION = '0.4.0'
```

Comment:
We haven't made a release, so can we do something like `0.4.0-experimental` or so?

Reply:
Done. Followed these recommendations: https://packaging.python.org/guides/distributing-packages-using-setuptools/#choosing-a-versioning-scheme